src/modules/ordersTable/pure/ReceiptModal/index.tsx

```diff
@@ -82,6 +91,19 @@ export function ReceiptModal({
           <CloseIcon onClick={() => onDismiss()} />
         </styledEl.Header>
 
+        {twapOrder && (
+          <styledEl.InfoBannerWrapper>
+            <InlineBanner
+              type="information"
+              content={
+                isTwapPartOrder
+                  ? `Part of a ${twapOrder.order.n}-part TWAP order split`
+                  : `TWAP order split into ${twapOrder.order.n} parts`
+              }
+            />
+          </styledEl.InfoBannerWrapper>
+        )}
```

Comment:
Not for this PR, just realized this now.

Isn't this violating the isolation principle?
The ordersTable component knows about specific order types.
Maybe it should instead be like TradeWidget and similar where the custom fields are consumed as slots?

Reply:
It's definitely should, my bad :(
Just don't have enough time, to refactor this module and add slots

Reply:
Created an issue so we don't forget https://github.com/cowprotocol/cowswap/issues/2671